parquet/src/main/java/org/apache/iceberg/parquet/Parquet.java

```diff
@@ -104,19 +104,21 @@
 import org.apache.parquet.column.ParquetProperties.WriterVersion;
 import org.apache.parquet.crypto.FileDecryptionProperties;
 import org.apache.parquet.crypto.FileEncryptionProperties;
-import org.apache.parquet.hadoop.ParquetFileReader;
-import org.apache.parquet.hadoop.ParquetFileWriter;
-import org.apache.parquet.hadoop.ParquetOutputFormat;
+import org.apache.parquet.hadoop.*;
```

Comment:
if you check compile you will have checkstyle error for this. Iceberg doesn't use * import.

Reply:
@ajantha-bhat yes I have fixed this in my local, fixing 1 Integration test case, then will push it again